contributing.md

```diff
@@ -0,0 +1,54 @@
+# Contributing
+
+Welcome to the Pelican project! We are excited to have you contribute to our open-source project. This guide will help you get started with setting up your development environment, understanding our coding standards, and making your first or next contribution.
+
+## Getting started
+
+To start contributing to Pelican Panel, you need to have a basic understanding of the following:
+
+* [PHP](https://php.net) & [Laravel](https://laravel.com)
+* [Livewire](https://laravel-livewire.com) & [Filament](https://filamentphp.com)
+* [Git](https://git-scm.com) & [Github](https://github.com)
+
+## Dev Environment Setup
+
+1. Fork the Repository
+2. Clone your Fork
+3. Install Dependencies (PHP modules & composer, and run `composer install`)
+4. Configure your Environment (via `php artisan p:environment:setup`)
+5. Set up your Database (via `php artisan p:environment:database`) and run Migrations (via `php artisan migrate --seed --force`)
+6. Create your first Admin User (via `php artisan p:user:make`)
+7. Start your Webserver (e.g. Nginx or Apache)
+
+As IDE we recommend [Visual Studio](https://visualstudio.microsoft.com)/ [Visual Studio Code](https://code.visualstudio.com) (free) or [PhpStorm](https://www.jetbrains.com/phpstorm) (paid).
```

Comment:
```suggestion
As IDE we recommend [Visual Studio Code](https://code.visualstudio.com) [Sublime Text](https://sublimetext.com) (free) or [PhpStorm](https://www.jetbrains.com/phpstorm) (paid).
```

I wouldn't recommend using Visual Studio to do anything web related so i'v added Sublime Text as an alternative to VSCode which is also free

Reply:
Visual Studio works fine for web development. But Sublime text is basically just a lite version of vscode.